README.rst

```diff
@@ -57,7 +57,7 @@ Then you can use the API like so:
 
 Tags will show up for you automatically in forms and the admin.
 
-``django-taggit`` requires Django 2.2 or greater.
```

Comment:
i think django 2.2 is still supported til april 2022

Reply:
I was about to keep Django 2.2 til April 2022. but here is Tim [comment](https://github.com/jazzband/django-hosts/pull/141#discussion_r766116557) about it:

> No, once Django 4.0 is out, it's time to drop support for all versions older than the most recent LTS. Otherwise, it could require ugly shims to add 4.0 support (which removes a lot of deprecated things). This is outlined in https://www.djangoproject.com/weblog/2015/jun/25/roadmap/.

Reply:
but for smoother upgrade path keeping compat little bit longer is good for users

Reply:
Thanks @auvipy for the review. I updated the PR and removed just Django 3.1

Reply:
great thanks